ENIGMAsystem/SHELL/Platforms/General/PFwindow.h

```diff
@@ -235,7 +235,7 @@ void window_set_position(int x, int y);
 void window_set_size(unsigned int width, unsigned int height);
 void window_set_rectangle(int x, int y, int width, int height);
 void window_center();
-void window_default(bool center_size);  // default false specified in platforms mandatory
+void window_default(bool center);  // default true specified in platforms mandatory
```

Comment:
Is centering the window the default behavior in GM, or is that what changed between versions?

Reply:
Centering the window is GM 8.1 behavior for window_default and on room start. GM studio stopped doing this at room start and dropped the window_default function.

Reply:
If GMS dropped the window_default function, and GM8.1 did not center, then why are you changing ENIGMA's to center the window?

Reply:
@RobertBColton GM 8.1 did center it, as i said, it did that with both window_default, and every room start. GMStudio does not have window_default, and does not center every room start.

Reply:
Interesting, perhaps it changed between 8.0 and 8.1 then because I recall testing it as well.